index.html

```diff
@@ -6,5 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
   </head>
-  <body></body>
+  <body>
+    This is a new test that i just added.
```

Comment:
I love this line of code, but it would be even better if everything was in upper case.